storage/plugins/viaf-authority/ViafAuthorityPlugin.php

```
@@ -0,0 +1,918 @@
+                'name'      => $term,
+                'nametype'  => (string) ($item['nametype'] ?? 'unknown'),
+                'viafUrl'   => 'https://viaf.org/viaf/' . $viafId,
+                'lc'        => (string) ($item['lc']   ?? ''),
+                'dnb'       => (string) ($item['dnb']  ?? ''),
+                'isni_id'   => $isniId,
+                'isni_uri'  => $isniUri,
+            ];
+        }
+
+        return $this->json($response, ['results' => $results]);
+    }
```

PHPMD (2.15.0)

[warning] The method viafSuggestAction() has a Cyclomatic Complexity of 17. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)